config/webpack.config.prod.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
       // in the main CSS file.
       {
         test: /\.css$/,
+        include: /(node_modules)|(-nomodules\.css$)/,
```

Comment:
@jniederm , I don't expect that.

I think we can achieve all potential customization of Patternfly (or so) on module level. 
Anyway, I don't expect any major "rework" of styling in general, just local adjustments for a component or it's children. At least this is my best wish :-)